README.md

```diff
@@ -166,7 +166,8 @@ There are two events that can be used as triggers for this action, each with dif
 
 ## Outputs
 
-In case the validation fails, this action will populate the `error_message` ouput.
+- In any case, this action will populate the `type`, `scope` and `subject` ouputs.
+- In case the validation fails, this action will populate the `error_message` ouput.
```

Comment:
"In any case" is not quite true, see [`src/index.js`](https://github.com/amannn/action-semantic-pull-request/blob/7dc0a564d2dfd88d2853ad8eb4000ee643ad2416/src/index.js).

I'd suggest this wording:

```suggestion
- The outputs `type`, `scope` and `subject` are populated, except for if the `wip` option is used.
- The `error_message` output will be populated in case the validation fails.
```